README.md

```diff
@@ -1 +1,10 @@
-# Hello-Microverse-Project
\ No newline at end of file
+# Hello-Microverse-Project
+ Hello Microverse
+
+ Built with:
+HTML,
+CSS.
+
+ Technologies used:
+Github,
+Vscode.
```

Comment:
- [ ] Kindly use [this template](https://github.com/microverseinc/readme-template) and adjust it for your project.

Reply:
Thank you.
I have gone through the corrections.
How does it look now?